src/autotype/mac/AutoTypeMac.cpp

```diff
@@ -488,3 +493,17 @@ void AutoTypeExecutorMac::execKey(AutoTypeKey* action)
     m_platform->sendKey(action->key, false);
     usleep(25 * 1000);
 }
+
+void AutoTypeExecutorMac::execClearField(AutoTypeClearField* action = nullptr)
+{
+    Q_UNUSED(action);
+
+    m_platform->sendKey(Qt::Key_Control, true, true);
+    m_platform->sendKey(Qt::Key_A, true, true);
+    m_platform->sendKey(Qt::Key_Control, false, true);
```

Comment:
I did in the first commit but it's not how OSX recognizes the key sequence in an unsimulated ⌘+A:

![image](https://cloud.githubusercontent.com/assets/107672/24366193/bc146062-12ee-11e7-9081-4c8bf99eb1bb.png)